app/assets/javascripts/darkswarm/all.js.coffee

```diff
@@ -1,4 +1,4 @@
-#= require jquery
+#= require jquery2
 #= require jquery_ujs
 #= require jquery.ui.all
```

Comment:
Does this one relies on jQuery 1 or 2 or whatever?

Reply:
They should be compatible :+1:

Reply:
Nice! 👌